src/main/java/com/tekenable/config/AuditConfigurator.java

```diff
@@ -10,26 +14,24 @@
 
```

Comment:
Ah that's fair enough but if we need to refactor it like this do that we will be able to unit test the expected battery of produced ddl.